<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinCode</class>
 <widget class="QWidget" name="PinCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>901</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="button1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button9">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>560</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter PIN-Code</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>260</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>410</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonBack">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>710</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonClear">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>710</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CLEAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button0">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>710</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>410</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>260</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>410</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonEnter">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>710</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ENTER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button8">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>560</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>560</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
